Makefile.COMMON

```diff
@@ -101,7 +97,7 @@ $(BINARY): $(GOCC) $(SRC) dependencies-stamp Makefile Makefile.COMMON
 archive: $(ARCHIVE)
 
 $(ARCHIVE): $(BINARY)
-	tar -czf $@ $<
+	tar --owner=root --group=root -czf $@ $<
```

Comment:
Well, what flags does the darwin/bsd version support?
On Sep 30, 2015 3:08 AM, "Fabian Reinartz" <notifications@github.com> wrote:

> In Makefile.COMMON
> <https://github.com/prometheus/consul_exporter/pull/12#discussion_r40764062>
> :
>
> > @@ -101,7 +97,7 @@ $(BINARY): $(GOCC) $(SRC) dependencies-stamp Makefile Makefile.COMMON
> >  archive: $(ARCHIVE)
> >
> >  $(ARCHIVE): $(BINARY)
> > -	tar -czf $@ $<
> > +	tar --owner=root --group=root -czf $@ $<
>
> Those do *not* work with OSX tar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/consul_exporter/pull/12/files#r40764062>.
>